src/main/kotlin/com/github/bzz/intellij/requests/Requester.kt

```diff
@@ -0,0 +1,36 @@
+package com.github.bzz.intellij.requests
+
+import com.github.bzz.intellij.com.github.bzz.intellij.requests.ServerResponse
+import com.google.gson.Gson
+import java.net.URI
+import java.net.http.HttpClient
+import java.net.http.HttpRequest
+import java.net.http.HttpResponse
+
+
+object Requester {
+    private val client = HttpClient.newBuilder().build()
+
+    fun getModelSuggestions(context: String, ip: String? = null, port: Int? = null): ServerResponse {
+        val request = HttpRequest.newBuilder()
+            .uri(URI.create("http://${ip ?: "localhost"}:${port ?: 8000}"))
+            .POST(HttpRequest.BodyPublishers.ofString(context))
+            .header("User-Agent", "Mozilla/5.0")
```

Comment:
Nice touch! Though probably no need to pretend a browser calling our own API.
May be having a plugin version as another header could be usefull for server-side logs, but probably that is too much hustle to implement ATM